internal/strace/strace.go

```diff
@@ -151,7 +174,30 @@ func (r *Result) recordCommand(cmd, env []string) {
 	}
 }
 
-func (r *Result) parseSyscall(syscall, args string) error {
+func (r *Result) parseEnterSyscall(syscall, args string) error {
+	switch syscall {
+	case "write":
+		match := writePattern.FindStringSubmatch(args)
+		// Example: django.template.base\nImporting django.template"..., 0xe64
+		writeBufAndBytesWritten := match[2]
+		for i := len(writeBufAndBytesWritten) - 1; i >= 0; i-- {
+			// TODO: A future PR will use the index right before the comma to get the end of the write buffer.
```

Comment:
Do you mean just using https://pkg.go.dev/strings#LastIndex ? 

That will indeed be a lot easier to read :)